Simplenote/SPNoteEditorViewController+Extensions.swift

```diff
@@ -0,0 +1,98 @@
+import UIKit
+
+// MARK: - History
+//
+extension SPNoteEditorViewController {
+
+    /// Indicates if note history is shown on screen
+    ///
+    @objc
+    var isShowingHistory: Bool {
+        return historyCardViewController != nil
+    }
+
+    /// Shows note history
+    ///
+    @objc
+    func showHistory() {
+        let viewController = newHistoryViewController()
+        viewController.present(from: self)
+    }
+
+    private func newHistoryViewController() -> SPCardViewController {
+        let loader = SPHistoryLoader(note: currentNote)
+        let controller = SPNoteHistoryController(note: currentNote, loader: loader)
+        let historyViewController = SPNoteHistoryViewController(controller: controller)
+        let cardViewController = SPCardViewController(viewController: historyViewController)
+
+        controller.delegate = self
+
+        historyLoader = loader
+        historyCardViewController = cardViewController
+
+        return cardViewController
```

Comment:
This API is a bit coupled: 

- Returns a `SPCardViewController` instance
- But also initializes `historyLoader` and `historyCardViewController` ivars

And those actions aren't explicited in the method signature. Do you think this could be cleared up a bit?

Reply:
Couldn't think of a good way to modify existing `newHistoryViewController`, but moved the code to `showHistory`, as that's the method to setup and configure everything related to history.